<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Voorspelling</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="left_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 203, 234);

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_left_1">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>278</y>
       <width>306</width>
       <height>126</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(105, 105, 105);</string>
     </property>
     <property name="text">
      <string>Voorspelling, aplicación de escritorio para aprendizaje de máquina supervisado</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_left_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>500</y>
       <width>306</width>
       <height>126</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(105, 105, 105);</string>
     </property>
     <property name="text">
      <string>Iván-David Rey-Rueda
buc16372023@mail.udes.edu.co

German Diaz-Figueredo
buc17372054@mail.udes.edu.co</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_left_3">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>730</y>
       <width>306</width>
       <height>126</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(105, 105, 105);</string>
     </property>
     <property name="text">
      <string>Universidad de Santander
Ingeniería de Software
2020</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_left_4">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>870</y>
       <width>306</width>
       <height>126</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(105, 105, 105);</string>
     </property>
     <property name="text">
      <string>Version 1.0
Licencia BSD 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="logo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>230</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/voorspelling_logo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="up_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1440</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 136, 224);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_tittle">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>941</width>
       <height>72</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>32</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PASO 2 - Cargar conjunto de datos</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="main_area">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>1030</width>
      <height>830</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 252, 255);
border: 4px solid;
border-color: rgb(225, 221, 235);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_description">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>690</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(105, 105, 105);
border: none;</string>
     </property>
     <property name="text">
      <string>Seleccione el formato de conjunto de datos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>560</y>
       <width>330</width>
       <height>130</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(154, 148, 238);
border-left-color: rgb(195, 190, 239);
border-top-color: rgb(195, 190, 239);
border-bottom-color: rgb(154, 148, 238);
border-right-color: rgb(154, 148, 238);
border-radius: 20px;
}
QPushButton:hover{
background-color: rgb(160, 155, 245);
}
QPushButton:pressed{
background-color: rgb(150, 140, 230);
border-left-color: rgb(140, 135, 220);
border-top-color: rgb(140, 135, 220);
border-bottom-color: rgb(195, 190, 239);
border-right-color: rgb(195, 190, 239);
}</string>
     </property>
     <property name="text">
      <string>Siguiente</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_info_data_type">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: rgba(101, 85, 198, 0.2);
border: 2px solid #6555C6;
box-sizing: border-box;
border-radius: 25px;


position: absolute;
width: 25px;
height: 44.55px;
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 30px;
line-height: 56px;
display: flex;
align-items: center;
text-align: center;

color: #6555C6;
}
QPushButton:hover{
background-color: rgb(190, 185, 250);
}
QPushButton:pressed{
background-color: rgb(160, 150, 240);
}</string>
     </property>
     <property name="text">
      <string>i</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-image: url(:/botton/bxs-left-arrow-circle.svg);
border-radius: 35px;
border: none;


color: #6555C6;
}
QPushButton:hover{
background-color: rgb(235, 225, 250);

}
QPushButton:pressed{

background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(200, 190, 250, 205), stop:0.6 rgba(200, 190, 250, 205), stop:1 rgba(255, 255, 255, 0));

}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_csv">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

background-color: rgb(154, 148, 238);
border-left-color: rgb(195, 190, 239);
border-top-color: rgb(195, 190, 239);
border-bottom-color: rgb(154, 148, 238);
border-right-color: rgb(154, 148, 238);
border-radius: 25px;

}
QPushButton:hover{
background-color: rgb(160, 155, 245);
}
QPushButton:pressed{
background-color: rgb(150, 140, 230);
border-left-color: rgb(140, 135, 220);
border-top-color: rgb(140, 135, 220);
border-bottom-color: rgb(195, 190, 239);
border-right-color: rgb(195, 190, 239);

}</string>
     </property>
     <property name="text">
      <string>CSV</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tsv">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgb(154, 148, 238);
border-left-color: rgb(195, 190, 239);
border-top-color: rgb(195, 190, 239);
border-bottom-color: rgb(154, 148, 238);
border-right-color: rgb(154, 148, 238);
border-radius: 25px;


}
QPushButton:hover{
background-color: rgb(160, 155, 245);
}
QPushButton:pressed{
background-color: rgb(150, 140, 230);
border-left-color: rgb(140, 135, 220);
border-top-color: rgb(140, 135, 220);
border-bottom-color: rgb(195, 190, 239);
border-right-color: rgb(195, 190, 239);

}</string>
     </property>
     <property name="text">
      <string>TSV</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_background">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 211, 230);
color: rgb(135, 135, 135);
border-radius: 25px;
border: none;</string>
     </property>
     <property name="text">
      <string>     TSV          CSV</string>
     </property>
    </widget>
    <zorder>lbl_background</zorder>
    <zorder>lbl_description</zorder>
    <zorder>btn_next</zorder>
    <zorder>btn_info_data_type</zorder>
    <zorder>btn_back</zorder>
    <zorder>btn_csv</zorder>
    <zorder>btn_tsv</zorder>
   </widget>
   <zorder>up_bar</zorder>
   <zorder>left_bar</zorder>
   <zorder>logo</zorder>
   <zorder>main_area</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Images_voor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_tsv</sender>
   <signal>clicked()</signal>
   <receiver>btn_csv</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>859</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_csv</sender>
   <signal>clicked()</signal>
   <receiver>btn_tsv</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_tsv</sender>
   <signal>clicked()</signal>
   <receiver>btn_tsv</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>859</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_csv</sender>
   <signal>clicked()</signal>
   <receiver>btn_csv</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
